R/expect_lint.R

```diff
@@ -93,24 +93,25 @@ expect_lint <- function(content, checks, ..., file = NULL, language = "en", igno
   }
 
   local({
-    itr <- 0L
+    itr_env <- environment()
```

Comment:
If you fully qualify everywhere, you can use `new.env(parent = emptyenv())` for safety.